Ports/cavestory/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="cavestory"
```

Comment:
All strings without interpolation should use single quotes. For example, in this case:

```
port='cavestory`
```